client/cmd/dexc/ui/config.go

```diff
@@ -72,6 +72,7 @@ type Config struct {
 	ReloadHTML bool   `long:"reload-html" description:"Reload the webserver's page template with every request. For development purposes."`
 	DebugLevel string `long:"log" description:"Logging level {trace, debug, info, warn, error, critical}"`
 	LocalLogs  bool   `long:"loglocal" description:"Use local time zone time stamps in log entries."`
+	TorProxy   string `long:"torproxy" description:"Connect via TOR (eg. 127.0.0.1:9050)."`
```

Comment:
Need to add TorProxyUsername and TorProxyPassword for them to show up on help `-h`, and also to be settable at all I think.

Reply:
I'm trying to wrap my mind around the use case for setting user/pass at dexc startup.  They dictate how the circuit is created, I believe.  If you set them, that makes tor use a specific circuit for dexc.  If not set, that uses whatever circuit it has? Or?

Reply:
I actually don't know how to set a password... I guess it is in the config file somewhere. May not be a useful setting?

Reply:
Apparently it's not actually used by the tor socks5 server to authenticate, but to specify circuit uniqueness.  At least I think.  Input from dhill would be good here.

Reply:
Essentially:  same user/pass = tor isolation off.  randomized user/pass each connection = tor isolation on.  or TorIsolation: yes causes go-socks to randomize the user/pass for you.

https://github.com/decred/go-socks/blob/master/socks/dial.go#L120

I will push a PR to remove User/Pass.. and just add a --torisolation bool.  Does that work?